Makefile

```diff
@@ -125,6 +125,9 @@ check:
 	@echo "errcheck"
 	@! errcheck -ignore 'bytes:Write.*,io:(Close|Write),net:Close,net/http:(Close|Write),net/rpc:Close,os:Close,database/sql:Close,github.com/spf13/cobra:Usage' $(PKG) | grep -vE 'yacc\.go:'
 	@echo "vet"
+	@! go tool vet . 2>&1 | \
```

Comment:
i think you want `go vet` here since it correctly ignores bogus directories like `.git`. See https://github.com/golang/go/issues/11727